charts/data-persistence-layer-bundle/values.yaml

```
@@ -0,0 +1,32 @@
+# #############################################################################
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# #############################################################################
+
+# Default values for data-persistence-layer-bundle
+# This is a YAML-formatted file.
+
+simple-data-backend:
+  nameOverride: dataprovider-submodelserver
+  ingress:
+    enabled: true
```

due to an issue with the simple-data-backend docker image, we need to set this manually to "main" right now. 
```suggestion
simple-data-backend:
  nameOverride: dataprovider-submodelserver
  image:
    tag: "main"
  ingress:
    enabled: true
```